### ISSUE TITLE
revisit service access strategy

### ISSUE THREAD

**Issue by joel-bluedata** [bluek8s/kubedirector#50]
May depend on issue #5. We probably want to have NodePort vs. LoadBalancer settable as a site default... and I'm not sure it's even useful to allow overriding it at deploy time.

Also we need to think about supporting Ingress. If that turns into a real project we can split it into a separate issue.

**Comment by joel-bluedata** [comment]
Split Ingress support to issue #113.
